Content.Server/Abilities/Psionics/PsionicAbilitiesSystem.cs

```diff
@@ -205,7 +208,7 @@ public void MindBreak(EntityUid uid)
         var ev = new OnMindbreakEvent();
         RaiseLocalEvent(uid, ref ev);
 
-        if (_config.GetCVar(CCVars.ScarierMindbreaking))
+        if (_config.GetCVar(CCVars.ScarierMindbreaking) || _tag.HasTag(uid,"TraitScaryMindbreaking" ))
```

Comment:
This really shouldn't be hardcoded to to do this. You should instead modify the OnMindbreakEvent to include a bool for ForceScarrierMindbreaking, which can be set by any component listening to the OnMindBreakEvent. Then you'll want to add a component for this mindbreaking vulnerability, which has a subscription to ScarierMindBreaking, and when it receives the event, sets said boolean to force it.